gatt_blacklist_policy.md

```diff
@@ -0,0 +1,82 @@
+# Policy for adding UUIDs to the GATT Blacklist
+
+## UUIDs are added through Github issues in this repository
+
+There is currently no minimum or maximum time between
+when the issue is filed and when the new UUID is added to the blacklist.
```

Comment:
Should we mention the finch blacklist that allows Chrome to deploy very quickly in case of an emergency?